### ISSUE TITLE
Upgrading EF Core 2.1.2.0 to 2.1.3.0 -  Dependent Assemblies in app.config are not all automatically updated

### ISSUE THREAD

**Issue by JRBonnema** [dotnet/efcore#13373]
I noticed that after installing the latest 2.1.3.0 version of EFCore the three packages below are not automatically updated to the new version. I have to do it by hand. Note that other packages are automatically updated. I didn't notice these behaviour during earlier updates, so I assume that this a new  problem. 

```
 <dependentAssembly>
    <assemblyIdentity name="Microsoft.EntityFrameworkCore" publicKeyToken="adb9793829ddae60" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-2.1.2.0" newVersion="2.1.2.0" />
  </dependentAssembly>
  <dependentAssembly>
     <assemblyIdentity name="Microsoft.EntityFrameworkCore.SqlServer" publicKeyToken="adb9793829ddae60" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-2.1.2.0" newVersion="2.1.2.0" />
  </dependentAssembly>
  <dependentAssembly>
     <assemblyIdentity name="Microsoft.EntityFrameworkCore.Relational" publicKeyToken="adb9793829ddae60" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-2.1.2.0" newVersion="2.1.2.0" />
  </dependentAssembly>
```

**Comment by ajcvickers** [comment]
@JRBonnema Can you be more specific about what you mean by "installing the latest 2.1.3 version?" Installing what specifically from where?

**Comment by JRBonnema** [comment]
@acjvickers: I Installed the three mentioned packages EFCore packages (version 2.1.3.0) from NuGet using the menuoption Manage NuGet Packages for Solution inside Visual Studio - version 15.8.4

**Comment by JRBonnema** [comment]
Update: I did the same upgrade on another few solutions after have installed Visual Studio - version 15.8.5. I noticed a difference between these two versions of Visual Studio, but it still not working like it was before.  

Further these errors are occurring in rather complex solutions, each having A WCF project in .Net Framework 4.72 using EF Core, calling a class library containing the datamodel in .Net Core itself also using EF Core. Further a second project in the solution of a Windows service in .Net Framework 4.72 embedding these WCF projects.

**Comment by ajcvickers** [comment]
@JRBonnema I'm still not fully understanding what you did and what the issue is. Are you saying that you installed 2.1.3 versions into your project, but that this didn't work and the versions in your project are still 2.1.2?

**Comment by JRBonnema** [comment]
@acjvickers The title says it all : Upgrading EF Core 2.1.2.0 to 2.1.3.0 - Dependent Assemblies in app.config are not all automatically updated. The upgrading itself is working fine. The Nuget packages are installed. The issue I have is that the dependent assemblies in  VS 15.8.4 were in three cases not updated. In VS 15.8.5 there is improvement, but in the case of the complex solutions I describe above I must still do 1 manual adjustment in app.config (for "Microsoft.EntityFrameworkCore") after upgrading. Before for instance when I migrated to EF Core 2.1.1 or EF Core  2.1.2 there was no need for these manual adjustments after upgrading. The error seems dependent on the version of VS. I noticed the error first in 15.8.4 when upgrading to EF Core 2.1.3 but it might have been there in earlier versions.
